src/applications/personalization/profile/components/personal-information/PersonalInformation.jsx

```diff
@@ -31,24 +30,45 @@ const getScrollTarget = hash => {
   return document.querySelector(hashWithoutLeadingEdit);
 };
 
-const PersonalInformation = ({
-  clearSuccessAlert,
-  hasUnsavedEdits,
-  hasVAPServiceError,
-  openEditModal,
-  shouldShowProfileLGBTQEnhancements,
-  showDirectDepositBlockedError,
-}) => {
+const PersonalInformation = () => {
   const lastLocation = useLastLocation();
-  useEffect(() => {
-    if (shouldShowProfileLGBTQEnhancements)
-      document.title = `Personal Information | Veterans Affairs`;
-    else document.title = `Personal And Contact Information | Veterans Affairs`;
 
-    return () => {
-      clearSuccessAlert();
-    };
-  }, []);
+  const showDirectDepositBlockedError = useSelector(
+    state => !!cnpDirectDepositIsBlocked(state),
+  );
+  const hasUnsavedEdits = useSelector(
+    state => state.vapService.hasUnsavedEdits,
+  );
+  const hasVAPServiceError = useSelector(hasVAPServiceConnectionError);
+  const shouldShowProfileLGBTQEnhancements = useSelector(
+    showProfileLGBTQEnhancements,
+  );
+
+  const directDepositIsAlwaysShowing = useSelector(
+    profileAlwaysShowDirectDepositDisplay,
+  );
+
+  const dispatch = useDispatch();
+  const clearSuccessAlert = useCallback(
+    () => dispatch(clearMostRecentlySavedField()),
+    [dispatch],
+  );
+
+  const openEditModal = useCallback(() => dispatch(openModal()), [dispatch]);
+
+  useEffect(
+    () => {
+      if (shouldShowProfileLGBTQEnhancements)
+        document.title = `Personal Information | Veterans Affairs`;
+      else
+        document.title = `Personal And Contact Information | Veterans Affairs`;
+
+      return () => {
+        clearSuccessAlert();
+      };
+    },
+    [clearSuccessAlert, shouldShowProfileLGBTQEnhancements],
```

Comment:
another area that could be potentially removed / moved to a parent based utility like my above comment... 🤔